tinacms/tinacms
# Fix: FormView buttons have default value

Should fix #1305 

To test: 

- Start up `demo-gatsby` and try to open the 'Authors' form. Before the changes, the site would break. Now default button values are being provided.

I wonder if there should be some cms-level 'buttons' extracted from the sidebar, given that various modals and screens use these save and reset strings.
